imperial_coldfront_plugin/tasks.py

```diff
@@ -0,0 +1,22 @@
+"""Plugin tasks."""
+
+import datetime
+
+from dateutil.relativedelta import relativedelta
```

Comment:
Not familiar with this. What's the difference between using a `relativedelta` and a `datetime.timedelta`?

Reply:
Changed to use `timedelta`